.pipelines/mac-ios-spm-extensions-validation-pipeline.yml

```diff
@@ -91,6 +91,8 @@
         export ORT_EXTENSIONS_IOS_POD_LOCAL_PATH="swift/${POD_ARCHIVE_EXT}"
 
         xcodebuild test -scheme onnxruntime-Package -destination 'platform=iOS Simulator,name=iPhone 14'
+
+        xcodebuild test -scheme onnxruntime-Package -destination 'platform=macosx'
```

Comment:
currently the macos platform testing command only applies to `.pipelines/mac-ios-spm-extensions-validation-pipeline.yml`. (only this pipeline consumes a dev version local extensions-c pod). after release, can add to the other two ci pipelins as well.

Reply:
Is there a task added to remind us to do this? Maybe add under 'Release specific 1.17'.

Reply:
yep, just added a task to the board.